test/server/index.html

```diff
@@ -1,5 +1,6 @@
 <html>
 <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+<script src="https://unpkg.com/axios-mock-adapter/dist/axios-mock-adapter.js"></script>
```

Comment:
This will also require a change in the backend SDK's integration test's index.html files.

Reply:
supertokens/supertokens-node#187